website/templates/trademark_detailview.html

```diff
@@ -0,0 +1,155 @@
+{% extends "base.html" %}
+{% load gravatar %}
```

Comment:
Do you use these template tags?

Reply:
Thanks I forget to look over them 
Removed in commit ab6e601efa5fca5e3af1f7a3ee3532c480c10a4d